app/Mail/ResetPassword.php

```diff
@@ -33,6 +33,6 @@ public function build()
         return $this->view('emails.resetPassword')
             ->from([Config::get("mail.addresses.aas")])
             ->to([$to])
-            ->subject('ANDERWIJS - Wachtwoord gereset');
+            ->subject('[Anderwijs] Wachtwoord gereset');
```

Comment:
I see this subject prefix in all emails. Maybe make it configurable and centralize is?